final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,267 @@
+import requests
+import argparse
+import json
+import logging
+import socket
+import bs4
+from bs4 import BeautifulSoup
+import CustomException
+
+
+log = logging.getLogger("Log")
+
+
+def is_connected() -> bool:
+    log.info("Checking internet connection...")
+    try:
+        socket.create_connection(("www.google.com", 80))
+    except OSError:
+        pass
```

Comment:
Do you really need this block if you trying to catch a broader exception a bit lower?